src/components/search/SearchResults.jsx

```diff
@@ -47,162 +47,20 @@ const SearchResults = () => {
   );
 
   return (
-    <Container sx={{ marginTop: 10, marginBottom: 10 }} maxWidth="xl">
-      <Grid container spacing={4}>
+    <Container className='searchbox_conatiner' >
+      <div className='searchbox'>
         {filteredProducts.length > 0 ? (
-          filteredProducts.map((product) => (
```

Comment:
reason behind removing all these code? contains icons like star and favorite that you removed as well

Reply:
Hey @dinxsh basically it was a design of a card component but we already have designed one product component so we should use this component so that there is uniformity among cards

Reply:
If you want I could comment them but they are not needed any more

Reply:
hey @dinxsh can you please review this

Reply:
ah alright, could you just add that heart icon back then? @nishant0708

Reply:
okay i will

Reply:
hey @dinxsh  i checked 
![image](https://github.com/MAVRICK-1/e-commerce_website/assets/101548649/80237d8f-cbb0-4453-9250-a9727c1ee4e8)
there is already heart icon present so i guess we dont need to add another one

Reply:
ok seems good